err_stash.py

```diff
@@ -551,6 +551,10 @@ def merge(self, msg, branch_text, force=False):
         projects = self.config['STASH_PROJECTS']
         if not projects:
             return '`STASH_PROJECTS` not configured. Use `!plugin config Stash` to configure it.'
+        
+        github_organizations = self.config['GITHUB_ORGANIZATIONS']
```

Comment:
Can we add a test please?

Reply:
Sure, I'll do it!

Reply:
Also plase review the README, I see the configuration for the bot is outdated.

Reply:
Done